setup.py

```diff
@@ -51,13 +52,32 @@ def _str2list(pkgstr, onlystr):
 libnl = pkgconfig('libnl-3.0')
 libnl['libs'].append('nl-route-3')
 
-# don't reformat this line, Makefile parses it
+with open('README.rst') as f:
+    long_description = f.read()
+
 setup(name='ethtool',
       version=version,
       description='Python module to interface with ethtool',
+      long_description=long_description,
+      
       author='Harald Hoyer, Arnaldo Carvalho de Melo, David Sommerseth',
       author_email='davids@redhat.com',
+      
+      maintainer='Lumír Balhar, Miro Hrončok, Charalampos Stratakis, Sanqui',
+      maintainer_email='python-maint@redhat.com',
+      
       url='https://github.com/fedora-python/python-ethtool',
+      license='GPLv2',
```

Comment:
there is no standard about what this should be, but I'd go with the [SPDX license identifier](http://spdx.org/licenses/): GPL-2.0